### DISCUSSION TITLE
Why is named token not recommended to use?

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#4] [Questions] [answered]
When we create a new NFT token we can use `create()` or `create_named_token()`. If we use `create_named_token(token_name)` we can later retrieve the token address by calling `create_token_address(creator_addr, collection_name, token_name), which is very handy when the token name is the same as `creator_addr`. But what's downside of doing this?

**Comment by 0x-j** [comment]
@0xaptosj

**Comment by angieyth** [comment]
@0xaptosj

**Reply by 0x-j** [reply]
nice i got the email notification

**Comment by 0x-j** [comment, marked as answer]
Found the answer on [move spider](https://movespiders.com/courses/modules/advancedobjects/lessonId/7).
tldr: named object cannot be deleted, it might lead to congestion cause many creating name object tx cannot be parallelized

**Reply by gregnazario** [reply]
Additionally, it's possible to have a name conflict between multiple tokens.
